client-misk/src/main/kotlin/app/cash/backfila/client/misk/BackfilaModule.kt

```diff
@@ -24,13 +26,16 @@ import misk.inject.KAbstractModule
 class BackfilaModule(
   private val config: BackfilaClientConfig,
   @Deprecated(message = "Multibind backfills instead")
-  private val backfills: List<KClass<out Backfill<*, *, *>>>? = null
+  private val backfills: List<KClass<out Backfill<*, *, *>>>? = null,
+  private val loggingSetupProvider: KClass<out BackfilaClientLoggingSetupProvider> =
+      BackfilaClientNoLoggingSetupProvider::class
 ) : KAbstractModule() {
   override fun configure() {
     bind<BackfilaClientConfig>().toInstance(config)
 
     bind<BackfilaClient>().to<RealBackfilaClient>()
     bind<BackfilaManagementClient>().to<RealBackfilaManagementClient>()
+    bind<BackfilaClientLoggingSetupProvider>().to(loggingSetupProvider.java)
```

Comment:
to me, it's weird to use kclass just to get the java class out

Reply:
Yeah, I should have cleaned this up.